rust/ast-generator/patches/rust-analyzer.patch

```diff
@@ -1,16 +1,19 @@
 diff --git a/xtask/src/codegen/grammar.rs b/xtask/src/codegen/grammar.rs
-index e7534582f2..8bc9237737 100644
+index e7534582f2..49c96f1be3 100644
 --- a/xtask/src/codegen/grammar.rs
 +++ b/xtask/src/codegen/grammar.rs
-@@ -3,6 +3,7 @@
+@@ -3,7 +3,9 @@
  //! Specifically, it generates the `SyntaxKind` enum and a number of newtype
  //! wrappers around `SyntaxNode` which implement `syntax::AstNode`.
  
+-#![allow(clippy::disallowed_types)]
 +#![allow(warnings)]
- #![allow(clippy::disallowed_types)]
++#![allow(clippy)]
++#![cfg_attr(any(), rustfmt::skip)]
  
  use std::{
-@@ -23,7 +24,7 @@ use crate::{
+     collections::{BTreeSet, HashSet},
```

Comment:
These added lines look a bit odd.

Reply:
I think this might be the understandable confusion of looking at the diff of a diff: these lines are context lines in the new diff, added because I added the top-level attributes (which have a `+`, these lines don't)

Reply:
ah yes indeed.